docs/docs/getting_started/quick_start.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Quick Start
+tags: []
+sidebar_position: 0
+---
+
+## Installation
+
+The easiest way to develop with Noir is using Nargo the CLI tool. It provides you the ability to start new projects, compile, execute and test Noir programs from the terminal.
+
+You can use `noirup` the installation script to quickly install and update Nargo:
+
+```bash
+curl -L noirup.dev | bash
+noirup
+```
+
+After installing Noir, we install a proving backend to work with our Noir programs.
+
+Proving backends provide you the abilities to generate proofs, verify proofs, generate smart contracts and more for your Noir programs.
+
+Different proving backends provide different tools for working with Noir programs, here we will use the [Barretenberg proving backend](https://github.com/AztecProtocol/aztec-packages/tree/master/barretenberg) developed by Aztec Labs as an example.
+
+You can use `bbup` the installation script to quickly install and update BB, Barretenberg's CLI tool:
+
+You can find the full list of proving backends compatible with Noir in Awesome Noir.
+```bash
+curl -L bbup.dev | bash
+bbup
+```
+
+For the full list of proving backends compatible with Noir, visit [Awesome Noir](https://github.com/noir-lang/awesome-noir/?tab=readme-ov-file#proving-backends).
+
+
+## Nargo
+
+Nargo provides the ability to initiate and execute Noir projects. Let's initialize the traditional `hello_world`:
+
+```sh
+nargo new hello_world
+```
+
+Two files will be created.
+
+- `src/main.nr` contains a simple boilerplate circuit
+- `Nargo.toml` contains environmental options, such as name, author, dependencies, and others.
+
+Glancing at _main.nr_ , we can see that inputs in Noir are private by default, but can be labeled public using the keyword `pub`. This means that we will _assert_ that we know a value `x` which is different from `y` without revealing `x`:
+
+```rust
+fn main(x : Field, y : pub Field) {
+    assert(x != y);
+}
+```
+
+To learn more about private and public values, check the [Data Types](./noir/concepts/data_types/index.md) section.
+
+### Compiling and executing
+
+We can now use `nargo` to generate a _Prover.toml_ file, where our input values will be specified:
+
+```sh
+nargo check
+```
+
+Let's feed some valid values into this file:
+
+```toml
+x = "1"
+y = "2"
+```
+
+We're now ready to compile and execute our Noir program. By default the `nargo execute` command will do both, and generate the `witness` that we need to feed to our proving backend:
+
+```sh
+nargo execute hello-witness
```

Comment:
```suggestion
nargo execute
```

Following https://github.com/noir-lang/noir/pull/6031, `nargo execute` now writes a witness file named after the Noir package by default